core/clr

```diff
@@ -1 +1 @@
-Subproject commit c50610b44daf5d3a4e5761b90f9f13606d102b5d
+Subproject commit 5229b26a3775fbb2eb6009220da376deea8fcac0
```

Comment:
This contained changes incompatible with the upstream PyTorch 2.7 release branch, breaking our nightly pytorch release builds: https://github.com/ROCm/TheRock/issues/1048.

To continue supporting that branch we'll need to cherry-pick at least https://github.com/pytorch/pytorch/commit/04bd7e6850e8efec77994963ffee87549555b9c3. Going forward we should switch to the ROCm backport and 2.8/main.